packages/medusa/src/services/__tests__/product-variant.js

```diff
@@ -718,33 +766,31 @@ describe("ProductVariantService", () => {
       expect(moneyAmountRepository.create).toHaveBeenCalledWith({
         variant_id: IdMap.getId("ironman"),
         region_id: IdMap.getId("cali"),
+        currency_code: "usd",
```

Comment:
self-review: for some reason, on master the currency_code did not appeared even though it is a non nullable column and the the old setRegionPrices was having the currency_code, do I miss something?